captum/_utils/gradient.py

```diff
@@ -423,7 +423,9 @@ def _forward_layer_eval_with_neuron_grads(
     evals in a dictionary protected by a lock, analogous to the gather implementation
     for the core PyTorch DataParallel implementation.
     """
-    grad_enabled = True if gradient_neuron_index is not None or grad_enabled else False
+    grad_enabled = (
+        True if gradient_neuron_selector is not None or grad_enabled else False
```

Comment:
nit: redundant if-else, `grad_enabled = gradient_neuron_selector is not None or grad_enabled`